TouchEffect.Droid/PlatformTouchEff.cs

```diff
@@ -18,8 +24,12 @@ public class PlatformTouchEff : PlatformEffect
     {
         public static void Preserve() { }
 
+        
         private TouchEff _effect;
         private bool _isHoverSupported;
+        private RippleDrawable _ripple;
+        private FrameLayout _viewOverlay;
+        public AView _view => Control ?? Container;
```

Comment:
make it private and fix naming according to to naming convention (Properties should be started from a capital letter without underscope)

private AView View => Control ?? Container;